### ISSUE TITLE
Theme Resources

### ISSUE THREAD

**Issue by psahalot** [Tarendai/WordPress-The-Right-Way#27]
Hi @Tarendai  
I don't see any section for Themes. I will be happy to build one.
There could be various chapters under themes:
1. Template Hierarchy 
2. Code Standards
3. Functions, Hooks, Filters (Themes specific)
4. Theme Customizer
5. Data validation and sanitization
6. Theme Translation / Internalization 
7. Copyright / License Attribution

This is all I could think as of now. But, I am sure there's lot more. 
I will be happy to start the section and contribute on various topics. 
Looking forward to your reply and confirmation.

Thanks!

**Comment by franz-josef-kaiser** [comment]
A _Themes_ chapter could have information about displaying images and strategies for that. It could also show how to register image sizes, change how to display images or alter their output in themes, etc. Another interesting topic would be menus and walkers. Those alone could fill a whole book.

**Comment by Tarendai** [comment]
My thinking is that we already have a [templates](http://www.wptherightway.org/en/templates/README.html) chapter, perhaps it can be broken up to include the relevant parts and renamed to Themes & Templates?

**Comment by franz-josef-kaiser** [comment]
Yes, I'd rename that. And make _Templates_  a sub chapter of _Themes_. There's much more to themes that needs covering than just templates.
